src/commands/create_command.py

```diff
@@ -0,0 +1,37 @@
+import sys
+import json
+import os
+from abstract_command import *
+from keys import Keys
+sys.path.append(os.path.join(str(os.getcwd()).replace("commands", '').replace("tests", '').replace("src", ''), "src", "output"))
+from exceptions import *
+
+
+class CreateCommand(AbstractCommand):
+    def __init__(self, schema_file):
+        self.schema_file = schema_file
```

Comment:
Why do you need to save first three variable as instance variable ?

Reply:
Good point

Reply:
The first two are initialized to be able to use them to in the validate because it doesn't get any arguments as in the parent class, for the third one I think you're right

Reply:
Why don’t pass them in validate ? Why validate as instance method ?
Does validate need an access to the instance variable ?

is validate called from outside ?

that’s why I recommend validate to be a static method takes those parameter. Remove it from ICommand

Reply:
If it will be used only inside a class, so we do we need to insure that ?

Reply:
Because the program need to check for the output and this works as a reminder